README.md

```diff
@@ -75,6 +75,55 @@ The dataclass can have fields of the base types: `int`, `float`, `str`, `bool`,
 - A union of multiple dataclasses, that in turn contain fields of supported types,
   which will be parsed in [Subparsers](#subparsers).
 
+## Overriding defaults from a file
+
+Pydargs can also consume values from a JSON- or YAML-formatted file. To enable
+this, pass `load_from_config=True` to the `parse` function, which will add a `--config-file`
+command line argument. If provided, the values from this file will override the defaults
+of the dataclass fields. Any command line arguments passed will override the
+defaults provided in the file.
+
+For example, with the following contents in `defaults.json`:
+
+```json
+{
+  "a": 1,
+  "b": "abc"
+}
+```
+
+then running this code
+
+```python
+from dataclasses import dataclass
+from pydargs import parse
+
+
+@dataclass
+class Config:
+  a: int = 2
+  b: str = "def"
+
+
+if __name__ == "__main__":
+  config = parse(Config, load_from_config=True)
```

Comment:
I think this variable name should relate to a file e.g. `load_defaults_from_file` or `load_from_config_file` or `load_from_file`

Reply:
I agree that `load_from_config_file` would be a better name. But as this argument only adds the argument, would `add_config_file_argument` or `allow_config_file` be better? Or do you have further suggestions?

Reply:
Done. `add_config_file_argument` it is